assets/css/main.css

```diff
@@ -0,0 +1,125 @@
+body {
+  font: 100% Helvetica, sans-serif;
+  background: url(/assets/images/bg.jpg) no-repeat center center fixed;
+  background-size: cover;
+  margin: 0;
+  padding: 0;
+}
+
+.menu-item {
+  margin-left: 3rem;
+  margin-right: 3rem;
+  font-size: 2rem;
+}
+
+.menu-list {
+  /* display: -webkit-box;
+  display: -ms-flexbox; */
+  display: flex;
+  list-style-type: none;
+  -webkit-box-pack: center;
+  -ms-flex-pack: center;
+  justify-content: center;
+  padding: 0.5rem;
+}
+
+.menu-link {
+  color: #dfddeedc;
+  text-decoration: none;
+  -webkit-transition: all 0.5s ease-in-out;
+  transition: all 0.5s ease-in-out;
+}
+
+.menu-link:hover {
+  color: #8d0517;
+}
+
+#myMenu {
+  width: 90%;
+}
+
+.img-meal {
+  width: 20rem;
+  border-radius: 5px;
+  -webkit-transition: width 0.7s;
+  transition: width 0.7s;
+}
+
+.img-meal:hover {
+  width: 22rem;
+}
+
+.meals {
+  /* display: -webkit-box;
+  display: -ms-flexbox; */
```

Comment:
- [ ] It's better to not commit commented codes, would you remove them? That will make your code to look much more cleaner